sklearn/preprocessing/tests/test_data.py

```diff
@@ -1588,3 +1591,27 @@ def test_fit_cold_start():
         # with a different shape, this may break the scaler unless the internal
         # state is reset
         scaler.fit_transform(X_2d)
+
+
+def test_boxcox_transform():
+    # Apply boxcox transform and check is if it applying individually on each feature 
+    X = np.array([[4, 2, 1], [1, 6, 3], [1, 5, 2], [3, 1, 3]])
+    Xtr = boxcox(X)
+    n_features = X.shape[1]
+    for feature in range(n_features):
+        assert_array_equal(stats.boxcox(X[:, feature])[0], Xtr[:, feature])
+        assert_array_equal(boxcox(X[:, feature].reshape(-1, 1)).ravel(),
+                           Xtr[:, feature])
+
```

Comment:
Sorry will run it. I thought it would be better to first make sure the implementation is as expected and later take care of the pep8 styles.